core/src/main/java/org/apache/accumulo/core/metrics/Metric.java

```diff
@@ -304,36 +308,49 @@ public String getDescription() {
     return description;
   }
 
-  public MetricCategory getCategory() {
-    return category;
+  public MetricDocSection getCategory() {
```

Comment:
What is the motivation behind changing this name? It seems like renaming it to reference the docs ties it too closely to the document generation context, potentially limiting its use in other parts of the code. This makes it seem specific to documentation purposes, even though the Metric enum is used elsewhere. It appears to mix responsibilities with the DocGen class.

Reply:
I thought that MetricCategory was too vague and didn't convey the manner in which the value was used. MetricCategory / MetricDocSection is only used in MetricsDocGen.